src/databricks/labs/remorph/reconcile/query_builder.py

```diff
@@ -139,6 +104,48 @@ def _get_layer_transform(transform_dict: dict[str, Transformation], column: str,
     def _get_column_expr(func, column_transformations: list[TransformRuleMapping]):
         return [func(transformation) for transformation in column_transformations]
 
+    @staticmethod
+    def _get_column_map(column, column_mapping) -> str:
+        return column_mapping.get(column).target_name if column_mapping.get(column) else column
```

Comment:
```suggestion
        return column_mapping.get(column, Column(target_name=column)).target_name
```
Use the second parameter of dict.get to return a default

Reply:
fixed.